core/src/main/java/io/lionweb/lioncore/java/self/LionCore.java

```diff
@@ -77,7 +77,7 @@ public static Language getInstance() {
       INSTANCE = new Language("LionCore.M3");
       INSTANCE.setID("-id-LionCore-M3");
       INSTANCE.setKey("LionCore-M3");
-      INSTANCE.setVersion("1");
+      INSTANCE.setVersion("2023.1");
```

Comment:
Should we refer to the constant defined in JsonSerialization or viceversa defining it here and refer to it from JsonSerialization?